pkg/controller/controller.go

```diff
@@ -162,14 +162,40 @@ func (h *Handler) syncLocalVolumeProvider(instance *localv1.LocalVolume) error {
 	o.Status.Generations = children
 	o.Status.ObservedGeneration = &o.Generation
 	o.Status.State = operatorv1.Managed
-	// TODO handle conditions here too
+	h.addCondition(o, true, "")
 	err = h.syncStatus(instance, o)
 	if err != nil {
 		return fmt.Errorf("error syncing status %v", err)
 	}
 	return nil
 }
 
+func (h *Handler) addFailureCondition(lv *localv1.LocalVolume, oldLv *localv1.LocalVolume, err error) error {
+	message := fmt.Sprintf("error syncing local storage : %+v", err)
+	h.addCondition(lv, false, message)
+	syncErr := h.syncStatus(oldLv, lv)
+	if syncErr != nil {
+		logrus.Errorf("error syncing condition : %v", syncErr)
+	}
+	return err
+}
+
+func (h *Handler) addCondition(lv *localv1.LocalVolume, success bool, message string) {
+	operatorCondition := operatorv1.OperatorCondition{
+		Type:               operatorv1.OperatorStatusTypeAvailable,
+		Status:             operatorv1.ConditionTrue,
```

Comment:
Should this condition be true even on error?

Reply:
I have redid this